api/features/models.py

```diff
@@ -559,15 +559,16 @@ def clone(
         clone = deepcopy(self)
         clone.id = None
         clone.uuid = uuid.uuid4()
-        clone.feature_segment = (
-            FeatureSegment.objects.get(
-                environment=env,
-                feature=clone.feature,
-                segment=self.feature_segment.segment,
+
+        if self.feature_segment:
+            # For now, we can only create a new feature segment if we are cloning to another environment
+            # TODO: with feature versioning, ensure that we clone regardless.
```

Comment:
Well, it will be fixed by merging this into my separate feature versioning branch here: https://github.com/Flagsmith/flagsmith/pull/2382/files#diff-b1e7142918052c12cbfc6cf1bbac536beba781e7b9ea4910b343227a1783d89aR566

Reply:
I've added a link to the PR